cluster/examples/kubernetes/ceph/admission-controller/deployment/webhook-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
I'm not a big fan of deep directory structure. I'd suggest including the yaml and the scripts all in the same `admission-controller` directory.